fondat/aws/ses.py

```diff
@@ -0,0 +1,80 @@
+from email import message_from_string
+from string import Template
+
+import logging
+
+from collections.abc import Iterable
+from fondat.aws import Client
+from fondat.resource import resource, mutation
+from fondat.security import Policy
+from fondat.data import datacls
+from email.utils import formataddr
+from typing import Dict, Union
+
+
+_logger = logging.getLogger(__name__)
+
+
+@datacls
+class EmailRecipient:
+    email_address: str
+    first_name: str
+    last_name: str
+
+    def recipient_format(self) -> str:
+
+        return formataddr((f"{self.first_name} {self.last_name}", self.email_address))
+
+
+def ses_resource(
+    *,
+    client: Client,
+    policies: Iterable[Policy] = None,
+):
+    """
+    Create SES resource.
+
+    Parameters:
+    • client: SES client object
+    • message_type: type of value transmitted in each message
+    • security: security policies to apply to all operations
+    """
+
+    if client.service_name != "ses":
+        raise TypeError("expecting SES client")
+
+    @resource
+    class EmailResource:
+        @mutation(policies=policies)
+        async def send(
+            self,
+            email_from: Union[str, EmailRecipient],
```

Comment:
From and to should just be `str`. In SMTP, `MAIL FROM` and `RCPT TO` only support raw addresses, not names. If names should be included, they should be embedded in the `From` and `To` headers in the message body.